### PR TITLE
feat(frontend): add BTC address to receive modal

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

Similar to ETH, we add the BTC address to the `Receive` modal.

### CODE REVIEW

File: src/frontend/src/lib/derived/address.derived.ts [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
 import { derived, type Readable } from 'svelte/store';
 
 export const addressNotLoaded: Readable<boolean> = derived([addressStore], ([$addressStore]) =>
 	isNullish($addressStore)
 );
 
+export const btcAddressMainnet: Readable<OptionBtcAddress> = derived(
+	[addressStore],
+	([$addressStore]) =>
+		$addressStore?.[BTC_MAINNET_TOKEN_ID] === null
```
Comment (peterpeterparker) [inline src/frontend/src/lib/derived/address.derived.ts:15]: We can maybe create a function to avoid code duplication?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/derived/address.derived.ts (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/derived/address.derived.ts:15]: done!

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/derived/address.derived.ts:15]: Sorry... in another PR #2066

**Pull request merged**